### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by Skefflock** [remotely-save/remotely-save#732]
### What happened?

On unstable network connection or moving between areas with different wifi networks plugin catches an error which brings no problem besides an error message in console, on the screen and jumping icon of Obsidian in Dock bar.

text of error message from Obsidian console:
plugin:remotely-save:234 Error: net::ERR_NETWORK_IO_SUSPENDED
    at SimpleURLLoaderWrapper.<anonymous> (node:electron/js2c/browser_init:2:49935)
    at SimpleURLLoaderWrapper.emit (node:events:513:28)
eval @ plugin:remotely-save:234

Please make this error less active in getting user attention, there's still nothing can be done but clicking OK in alert message.
Thanks

### What OS are you using?

macOS

### What remote cloud services are you using? (Please choose the specified one if it's in the list)

OneDrive for personal

### Version of the plugin

0.5.12

### Version of Obsidian

1.6.5

### Using password or not

- [ ] Yes.

### Ensure no sensitive information

- [X] I ensure that no sensitive information is submitted in the issue.

**Comment by Skefflock** [comment]
By the way I've got error message level set to debug. Don't know if setting it to info would work.

**Comment by fyears** [comment]
so do you just want to avoid "on the screen and jumping icon of Obsidian in Dock bar."?

if you use auto sync, the notice will not show up. if you use manual sync, the notice will show up everytime. that's by design.

**Comment by Skefflock** [comment]
I do use autosync and this error message appears from time to time. It's not a notification about plugin workflow, but an alert message with ok button that shows up every day with some info about java script runtime error. Don't think it's designed this way.

**Comment by fyears** [comment]
do you have any screenshots? and does it run normal when network connection is ok?

**Comment by Skefflock** [comment]
I'll try to catch one next time it'll happen. It is run normal after I close alert message and connect to any network.

**Comment by Skefflock** [comment]
<img width="342" alt="изображение" src="https://github.com/remotely-save/remotely-save/assets/33117791/801f5257-525a-4142-b23d-ba80e0df715a">

Here's the alert I've been talking about.

**Comment by fyears** [comment]
but may i ask why do you think it's caused by the remotely save plugin? and if possible to reproduce then next time please open the console and check the console output

https://github.com/remotely-save/remotely-save/blob/master/docs/how_to_debug/check_console_output.md

**Comment by Skefflock** [comment]
O, I've already posted console output in the very first message. Please check it up.


> text of error message from Obsidian console: plugin:remotely-save:234 Error: net::ERR_NETWORK_IO_SUSPENDED at SimpleURLLoaderWrapper. (node:electron/js2c/browser_init:2:49935) at SimpleURLLoaderWrapper.emit (node:events:513:28) eval @ plugin:remotely-save:234

**Comment by fyears** [comment]
any more context like the message before the error? i would like to know on what stage is the plugin running

**Comment by Skefflock** [comment]
startting sync.
plugin:remotely-save:227 checkProRunnableAndFixInplace
plugin:remotely-save:227 checking "feature-google_drive", serviceType=onedrive
plugin:remotely-save:227 checking "feature-onedrive_full", serviceType=onedrive
plugin:remotely-save:227 checking "feature-box", serviceType=onedrive
plugin:remotely-save:227 checking "feature-pcloud", serviceType=onedrive
plugin:remotely-save:227 checking "feature-yandex_disk", serviceType=onedrive
plugin:remotely-save:227 checking "feature-koofr", serviceType=onedrive
plugin:remotely-save:227 checking "feature-azure_blob_storage", serviceType=onedrive
plugin:remotely-save:227 getJson, theUrl=https://graph.microsoft.com/v1.0/drive/special/approot/children
plugin:remotely-save:227 Onedrive accessToken updated
2plugin:remotely-save:227 getJson, theUrl=https://graph.microsoft.com/v1.0/drive/special/approot:/
plugin:remotely-save:227 getJson, theUrl=https://graph.microsoft.com/v1.0/drive/special/approot:/
plugin:remotely-save:227 getJson, theUrl=https://graph.microsoft.com/v1.0/drive/special/approot:/
plugin:remotely-save:227 getJson, theUrl=https://graph.microsoft.com/v1.0/drive/special/approot:/
plugin:remotely-save:227 getJson, theUrl=https://graph.microsoft.com/v1.0/drive/special/approot:/
plugin:remotely-save:227 getJson, theUrl=https://graph.microsoft.com/v1.0/drive/special/approot:/
plugin:remotely-save:234 Error: net::ERR_TIMED_OUT
    at SimpleURLLoaderWrapper.<anonymous> (node:electron/js2c/browser_init:2:49935)
    at SimpleURLLoaderWrapper.emit (node:events:513:28)
eval @ plugin:remotely-save:234
plugin:remotely-save:227 endding sync.


I've removed a part of onedrive url. Dont't think it has any meaning in this context.

**Comment by fyears** [comment]
Can you reproduce the error on Windows?

I do not have a mac, but on windows the error is caught properly

**Comment by Skefflock** [comment]
Unfortunately I can't. Don't have Windows installed on my laptop. I never use it. Is there any additional actions I should do to give you better information? Does plugin writes error logs somewhere?

**Comment by fyears** [comment]
your console log is sufficient, and almost like mine but on windows the error is caught so no system alter is shown. so i am stuck here. maybe i can find out a linux and try agin

**Comment by tanguoming** [comment]
same here. I confirm it's caused by remotely-save by disabling any other plugins.

MacOS / Onedrive personal

alert is like:
A JavaScript error occurred in the main process
Uncaught Exception:
Error: net::ERR_NETWORK_IO_SUSPENDED
at SimpleURLLoaderWrapper.<anonymous> (node:electron/js2c/browser_init:2:114352)
at SimpleURLLoaderWrapper.emit (node:events:519:28)

**Comment by fyears** [comment]
> same here. I confirm it's caused by remotely-save by disabling any other plugins.
> 
> 
> 
> MacOS / Onedrive personal
> 
> 
> 
> alert is like:
> 
> A JavaScript error occurred in the main process
> 
> Uncaught Exception:
> 
> Error: net::ERR_NETWORK_IO_SUSPENDED
> 
> at SimpleURLLoaderWrapper.<anonymous> (node:electron/js2c/browser_init:2:114352)
> 
> at SimpleURLLoaderWrapper.emit (node:events:519:28)

@tanguoming do you have windows or linux and can you reproduce the error?

**Comment by fyears** [comment]
![1722153942721](https://github.com/user-attachments/assets/7e668c04-5ebe-4654-9410-6960d241fd93)

Does the console error the same as yours?

And as you can see, my Obsidian (on Linux and Windows) shows up the error as a notice, not a pop up.

**Comment by Skefflock** [comment]
No, I've met pop up alert message with "ok" button. But just now I've realized I haven't seen it for a while. I opened console and the error is there. More than 10 times from last start. I have a hypotheses that the popup appear only when "Alert Notice Level"  is set to debug in plugin configs. Right now I have it set to info.

I'm going to test it reverting to debug level. Will write back.

**Comment by fyears** [comment]
ok please try `console.debug()` and `console.error()`

**Comment by Skefflock** [comment]
And today I've catched the error again after a week or two without. With this settings.
<img width="769" alt="изображение" src="https://github.com/user-attachments/assets/173d7bb9-f25d-4677-ba27-46f77a87ec18">

these were before yersterday with no error popups.
<img width="759" alt="изображение" src="https://github.com/user-attachments/assets/3900e8cb-387c-4300-8a11-bbf2183c04a3">

console.debug() and console.error() both return undefined

So, I may assume plugin debug level causes popup to appear, when info level supresses the error.

**Comment by Skefflock** [comment]
Finally, I'm not saying that any alert level value makes any difference. The popup appears spontaneously, no matter what.
